packages/shared/lib/core/network/utils/getTxOptions.ts

```diff
@@ -0,0 +1,11 @@
+import { ChainId } from '@core/network'
+import { Common } from '@ethereumjs/common'
+import { TxOptions } from '@ethereumjs/tx'
+
+export function getTxOptions(chainId: ChainId): TxOptions {
```

Comment:
- We should not use shortened names / words per the coding conventions, i.e. this should be `getTransactionOptions`
- Would be nice creating a wrapper type around `TxOptions` so that the previous rule isn't broken but mainly it will be easier to change later should we need / want to change the underlying data object we're using.